docs/api.rst

```diff
@@ -536,10 +536,59 @@ Transport exceptions
 - ``MessageReceiveError``: The transport encountered any other error while trying to receive a message.
 
 
-ASGI Transport
-++++++++++++++
+Redis Gateway Transport
++++++++++++++++++++++++
+
+The ``transport.redis_gateway`` module provides a transport implementation that uses Redis (in simple or Sentinel mode)
+for sending and receiving messages. This is the recommended transport for use with ``pysoa``, as it provides a
+convenient and performant backend for asynchronous service requests.
+
+Standard and Sentinel modes
+***************************
+
+The Redis Gateway transport has two primary modes of operation: in "standard" mode, the channel layer will connect to a
+specified list of Redis hosts, while in "Sentinel" mode, the channel layer will connect to a list of Sentinel hosts and
+use Sentinel to find its Redis hosts.
+
+Configuration
+*************
+
+The Redis Gateway transport takes the following extra keyword arguments for configuration:
+
+- ``backend_type``: Either "redis.standard" or "redis.sentinel" to specify which Redis backend to use (required)
+
+- ``backend_layer_kwargs``: A dictionary of arguments to pass to the backend layer
+
+  + ``connection_kwargs``: A dictionary of arguments to pass to the underlying Redis client
+
+  + ``hosts``: A list of strings (host names / IP addresses) or tuples (host names / IP addresses and ports) for Redis hosts or sentinels to which to connect (will use "localhost" by default)
```

Comment:
What is the timeout used when we attempt to connect?

Reply:
This can be specified in `connection_kwargs` and is a detail left up to the documentation for the Redis-Py library (I have updated this to reference the Redis-Py library).